index.html

```diff
@@ -3,7 +3,7 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
-
+  <meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
the meta viewport tag is key for getting your media queries to work.  Good job adding it.